warehouse/legacy/api/xmlrpc/views.py

```diff
@@ -28,15 +30,29 @@
     Role, Project, Release, File, JournalEntry, release_classifiers,
 )
 
-pypi_xmlrpc = functools.partial(
-    xmlrpc_method,
-    endpoint="pypi",
+_xmlrpc_method_partial = functools.partial(
```

Comment:
This might be cleaner by just eliminating the partial and inlining these options into the one method now.